tests/baselines/reference/asyncFunctionDeclaration15_es6.errors.txt

```diff
@@ -1,12 +1,8 @@
-tests/cases/conformance/async/es6/functionDeclarations/asyncFunctionDeclaration15_es6.ts(6,16): error TS1055: Type '{}' is not a valid async function return type.

```

Comment:
These errors will remain as we are keeping the logic that uses them for use with downlevel async functions for ES5 and earlier.